cmd/tools/rest/client.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const (
 	// DefaultTimeout should be > than ONTAP's default REST timeout, which is 15 seconds for GET requests
-	DefaultTimeout = 30
```

Comment:
In a few places, we've changed these to durations instead of unit-less ints. Might be worth doing the same here